api/v1alpha1/dynakube_types.go

```diff
@@ -23,12 +23,16 @@ type DynaKubeSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="API and PaaS Tokens",order=2,xDescriptors="urn:alm:descriptor:io.kubernetes:Secret"
 	Tokens string `json:"tokens,omitempty"`
 
+	// Namespace selector for Operator to know in which namespaces should it monitor applications
```

Comment:
```suggestion
	// Namespace selector for the Operator to know in which namespaces it should monitor applications. 
	// If no namespace selector is defined, it will monitor all namespaces.
```